girder_tech_journal/__init__.py

```diff
@@ -36,13 +36,52 @@
 from girder.models.user import User
 from girder.utility import mail_utils
 from girder.utility.model_importer import ModelImporter
+from girder.plugin import GirderPlugin
 from girder import events
 from girder_worker_utils.transforms.girder_io import GirderUploadToItem
 from tech_journal_tasks.tasks import processGithub, surveySubmission
 from . import constants
 from .models.journal import download_statistics, Journal
 
 
+class TechJournalPlugin(GirderPlugin):
+    DISPLAY_NAME = 'Tech Journal'
+    CLIENT_SOURCE_PATH = 'web_client'
+
+    def load(self, info):
+        techJournal = TechJournal()
```

Comment:
I think you should be fearless about moving the entire `TechJournal` class to its own file at `api/tech_journal.py`.